crates/node-core/src/events/node.rs

```diff
@@ -223,13 +223,21 @@ impl<DB> NodeState<DB> {
                     (self.safe_block_hash != Some(safe_block_hash) &&
                         self.finalized_block_hash != Some(finalized_block_hash))
                 {
-                    info!(
-                        ?head_block_hash,
-                        ?safe_block_hash,
-                        ?finalized_block_hash,
-                        ?status,
-                        "Forkchoice updated"
-                    );
+                    if status == ForkchoiceStatus::Invalid {
+                        info!(
+                            ?head_block_hash,
+                            ?safe_block_hash,
+                            ?finalized_block_hash,
+                            "Received invalid forkchoice updated message"
+                        );
+                    } else if status == ForkchoiceStatus::Syncing {
+                        info!(
+                            ?head_block_hash,
+                            ?safe_block_hash,
+                            ?finalized_block_hash,
+                            "Received forkchoice updated message when syncing"
+                        );
+                    }
```

Comment:
we need to handle all cases of `ForkchoiceStatus`, should use a match for this

Reply:
Done.